src/test/java/lotto/domain/LottoCashierTest.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LottoCashierTest {
+
+    private final LottoCashier lottoCashier = new LottoCashier();
+
+    @ParameterizedTest
+    @CsvSource(value = {"1000,1", "1500,1"})
+    void pay(int money, int lottoCount) {
+        List<Lotto> lottos = lottoCashier.pay(money);
+        assertThat(lottos.size()).isEqualTo(lottoCount);
+    }
```

Comment:
@DisplayName 을 통해 테스트의 설명을 작성해주면 어떨까요? 

hasSize 를 활용해봐도 좋을 것 같아요! 😄 
```suggestion
    @ParameterizedTest
    @CsvSource(value = {"1000,1", "1500,1"})
    void pay(int money, int lottoCount) {
        List<Lotto> lottos = lottoCashier.pay(money);
        assertThat(lottos).hasSize(lottoCount);
    }
```